Extensions/Guardrail+Settings.swift

```diff
@@ -34,29 +34,30 @@ public extension Guardrail where Value == HKQuantity {
         .max()!
     }
     
+    // Static "unconstrained" constant values before applying constraints
+    static let unconstrainedWorkoutCorrectionRange = Guardrail(absoluteBounds: 85...250,
+                                                               recommendedBounds: correctionRange.recommendedBounds.lowerBound.doubleValue(for: .milligramsPerDeciliter)...180,
+                                                               unit: .milligramsPerDeciliter)
+    
     fileprivate static func workoutCorrectionRange(correctionRangeScheduleRange: ClosedRange<HKQuantity>,
                                                    suspendThreshold: GlucoseThreshold?) -> Guardrail<HKQuantity> {
-        // Static "unconstrained" constant values before applying constraints
-        let workoutCorrectionRange = Guardrail(absoluteBounds: 85...250,
-                                               recommendedBounds: correctionRange.recommendedBounds.lowerBound.doubleValue(for: .milligramsPerDeciliter)...180,
-                                               unit: .milligramsPerDeciliter)
-        
         let absoluteLowerBound = [
-            workoutCorrectionRange.absoluteBounds.lowerBound,
+            unconstrainedWorkoutCorrectionRange.absoluteBounds.lowerBound,
             suspendThreshold?.quantity
         ]
         .compactMap { $0 }
         .max()!
         let recommmendedLowerBound = max(absoluteLowerBound, correctionRangeScheduleRange.upperBound)
         return Guardrail(
-            absoluteBounds: absoluteLowerBound...workoutCorrectionRange.absoluteBounds.upperBound,
-            recommendedBounds: recommmendedLowerBound...workoutCorrectionRange.recommendedBounds.upperBound
+            absoluteBounds: absoluteLowerBound...unconstrainedWorkoutCorrectionRange.absoluteBounds.upperBound,
+            recommendedBounds: recommmendedLowerBound...unconstrainedWorkoutCorrectionRange.recommendedBounds.upperBound
```

Comment:
are there tests for these changes?

Reply:
Strictly speaking...yes.  This file's only change was to "expose" already-established local vars as (internal) vars. I should have annotated the PR with that...